apl-core/src/test/resources/account_control_phasing.csv

```diff
@@ -1,5 +1,5 @@
 ACCOUNT_ID(-5|19|0),WHITELIST(2003|0|0),VOTING_MODEL(-6|3|0),QUORUM(-5|19|0),MIN_BALANCE(-5|19|0),HOLDING_ID(-5|19|0),MIN_BALANCE_MODEL(-6|3|0),MAX_FEES(-5|19|0),MIN_DURATION(5|5|0),MAX_DURATION(5|5|0),HEIGHT(4|10|0)
 7995581942006468815, null, 0, 1, null, null, 0, 300000000, 12, 113, 500
-2728325718715804811,(-8446656647637444484,),0,1,null,null,0,300000000,12,113,1000
--8446384352342482748,(-8446656647637444484,1344527020205736624,),0,1,null,null,0,300000000,12,113,2000
--4013722529644937202,(-8446656647637444484,1344527020205736624,-6724281675870110558,),0,1,null,null,0,300000000,12,113,3000
+2728325718715804811,(-8446656647637444484),0,1,null,null,0,300000000,12,113,1000
```

Comment:
Comma was left here due to h2 bug in Parser, it forces h2 to interpret single value as array. Is it alredy fixed?

Reply:
now this (-8446656647637444484,1344527020205736624,) throws an exception, so the last comma must not be there